style.css

```diff
@@ -1 +1,347 @@
+    color: inherit;
+}
+
+nav a {
+    text-decoration: none;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0;
+}
+img {
+    display: inline-block;
+    max-width: 100%;
+    height: auto;
+}
+
+p {
+    font-size: 1.07rem;
+    line-height: 1.8;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.section_title {
+    font-size: 3rem;
+    margin-bottom: 1rem;
+    font-weight: 500;
+}
+
+.section_subtitle {
+    font-size: 2rem;
+    margin-bottom: 1rem;
+    font-weight: 500;
+}
+
+.wrapper {
+    display: grid;
+    min-height: 100vh;
+    grid-template-areas:
+        "header"
+        "welcome"
+        "about"
+        "gallery"
+        "footer";
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(5, auto);
+}
+
+section {
+    padding-top: 10px;
+    padding-bottom: 10px;
+}
+
+/*    HEADER    */
+.header {
+    grid-area: header;
+    padding: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.content{
+    padding: 20px;
+}
+
+.main {
+    grid-area: main;
+    flex-grow: 1;
+}
+
+.menu_icon{
+    display: flex;
+    flex-direction: column;
+    row-gap: 5px;
+    align-items: center;
+    justify-content: center;
+    z-index: 101;
+    position: relative;
+}
+
+.menu_icon:hover{
+    cursor: pointer;
+}
+
+.menu_icon span{
+    width: 22px;
+    height: 2px;
+    background-color: var(--text);
+}
+
+.menu_list {
+    display: none;
+    column-gap: 2rem;
+    align-items: center;
+    justify-content: flex-end;
+    padding: 0 2rem;
+}
+
+.menu_item a {
+   text-transform: uppercase;
+    font-weight: 500;
+}
+
+.menu_item:hover a {
+    text-decoration: underline;
+}
+
+.menu_item.active a {
+    text-shadow: 1px 2px var(--yellow);
+}
+
+.welcome {
+    grid-area: welcome;
+    background-image: url("images/welcome_bg.jpeg");
+    background-repeat: repeat;
+    border-top: 2px solid var(--yellow);
+    border-bottom: 2px solid var(--yellow);
+}
+
+.content--welcome {
+    display: flex;
+    align-items: flex-end;
+    column-gap: 16px;
+}
+
+.welcome_img {
+    background-image: url("images/welcome.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: left center;
+    width: 100%;
+    min-width: 50px;
+    max-width: 60%;
+    height: 100%;
+    min-height: 200px;
+}
+
+.welcome_title{
+    font-family: var(--ff-secondary);
+    font-size: 2rem;
+    font-weight: 800;
+    line-height: 1.2;
+    text-shadow: 1px 1px 2px var(--bg);
+}
+
+.about {
+    grid-area: about;
+}
+
+
+.about_img {
+    height: 50vh;
+    background-image: url("images/gallery_L.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.about p {
+    margin-bottom: 2rem;
+}
+
+.gallery {
+    grid-area: gallery;
+}
+
+.gallery_list {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-column-gap: 20px;
+    grid-row-gap: 20px;
+}
+
+.footer {
+    border-top: 1px solid var(--yellow);
```

Comment:
This is a very fine line and may be difficult for users with low vision to see, especially with such a similar background color.

This is a really handy tool for checking if foreground-background color contrast is sufficient for all users to perceive.